Core/Core/Features/ActAsUser/ActAsUserWindow.swift

```diff
@@ -81,6 +84,19 @@ public class ActAsUserWindow: UIWindow {
             userInfo: ["style": traitCollection.userInterfaceStyle]
         )
     }
+
+    private var isTopAnOverlayException: Bool {
```

Comment:
I would name this based on what it does: it determines if there's a system picker being presented. My suggestion would be:`isPresentingSystemPicker`. This way when I read the code I can easily understand that if a system picker is being presented we don't do some things but on the other hand this exception naming doesn't tell me about the issue, exception from what exactly? I have to dig into the details of the implementation to understand what this checks.